FatPkg/EnhancedFatDxe/DiskCache.c

```diff
@@ -8,6 +8,229 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #include "Fat.h"
 
+/**
+  Helper function to check if any LBA is dirty in this cache line.
+
+  @param[in]    CacheTag   - CacheTag to check
+
+  @retval       TRUE       - Cache is Dirty
+                FALSE      - Cache is not Dirty
+
+**/
+STATIC
+BOOLEAN
+IsCacheTagDirty (
+  IN CACHE_TAG  *CacheTag
+  )
+{
+  if (CacheTag == NULL) {
+    ASSERT (CacheTag != NULL);
+    return FALSE;
+  }
+
+  return CacheTag->Dirty;
+}

```

Comment:
I prefer to remove this function for less layers and better readability.

Reply:
I’m fine to do that, this function was more complex and changed to this way at @mdkinney’s request, so I just want to make sure we aren’t flip flopping review comments.